### PR TITLE
Change VS insertions to use dotnet-bot account

### PR DESCRIPTION
**Pull request by JoeRobich**
Test run: https://dev.azure.com/dnceng/internal/_build/results?buildId=1734676&view=results (microsoft)

*Only needs to get past AzDO authorization to validate the changes. PR may not generate because it targets the wrong VS branch.

Also backports #60712

### CODE REVIEW

File: azure-pipelines-official.yml [hunk 1]
```diff
@@ -55,7 +54,12 @@ variables:
   - name: _DevDivDropAccessToken
     value: $(dn-bot-devdiv-drop-rw-code-rw)
   - group: ManagedLanguageSecrets
-
+  - group: DotNet-Roslyn-Insertion-Variables
+  - name: _DevDivAccessToken
+    value: $(dn-bot-devdiv-build-e-code-full-release-e-packaging-r)
+  - name: _DncEngAccessToken
```
Comment (JoeRobich) [inline azure-pipelines-official.yml:60]:
```suggestion
  - name: _DevDivInsertionAccessToken
    value: $(dn-bot-devdiv-build-e-code-full-release-e-packaging-r)
  - name: _DncEngInsertionAccessToken
```

File: azure-pipelines-official.yml [hunk 2]
```diff
@@ -328,8 +332,10 @@ stages:
 
     - template: eng/pipelines/insert.yml
       parameters:
-        clientId: $(ClientId)
-        clientSecret: $(ClientSecret)
+        buildUserName: "dn-bot@microsoft.com"
+        buildPassword: $(_DevDivAccessToken)
+        componentUserName: "dn-bot@microsoft.com"
+        componentPassword: $(_DncEngDropAccessToken)
```
Comment (JoeRobich) [inline azure-pipelines-official.yml:338]:
```suggestion
        buildPassword: $(_DevDivInsertionAccessToken)
        componentUserName: "dn-bot@microsoft.com"
        componentPassword: $(_DncEngInsertionAccessToken)
```

File: azure-pipelines-official.yml [hunk 3]
```diff
@@ -328,8 +332,10 @@ stages:
 
     - template: eng/pipelines/insert.yml
       parameters:
-        clientId: $(ClientId)
-        clientSecret: $(ClientSecret)
+        buildUserName: "dn-bot@microsoft.com"
+        buildPassword: $(_DevDivInsertionAccessToken)
```
Comment (RikkiGibson) [inline azure-pipelines-official.yml:336]: I would quote these, at least to put my paranoia at bay.

Comment (JoeRobich) [reply azure-pipelines-official.yml]: Added quotes and spun a new build

**Pull request closed without merging**